### DISCUSSION TITLE
How to generate yearly planner for Boox Max Lumi 2 / TabX  ( A4 size )

### DISCUSSION THREAD

**Discussion by harireghu** [kudrykv/latex-yearly-planner#74] [Q&A]
What changes do I need to do to generate yearly planner for Boox Max Lumi 2 / TabX  ?
These are A4 sized e-ink device more specifically their website says it is a 13.3" device.

I have a [Tab X](https://shop.boox.com/products/tabx) and  below are it's measurements ( taken by myself  ) :
The display measures 7.9 x 10.625 inches. 
The Notes App takes 0.5 inches on the left and 0.625 inches on the top for it's icons / buttons  etc.

I setup the tool to generate the existing planners , so I now have an environment where I can play around.
Just changing the page sizes [here](https://github.com/kudrykv/latex-yearly-planner/blob/main/cfg/base.yaml#L7 ) doesn't seem to work ( Generates a page with spaces at the bottom or has some other formatting issues attaching one page at the bottom ) 

Would appreciate some guidance on what to change and should I expect this to work .

Also how do I generate planner with lines instead of dots ?

Thanks!
-Hari

![image](https://user-images.githubusercontent.com/28494629/219931473-afea4dbf-5b9a-4916-ad1a-e320b742db13.png)

**Comment by kudrykv** [comment]
Hi @harireghu!

You also need to look into `cfg` dir, some config files. For example, `base.yaml`, the `quarterlylines` field. You should bump that number, so you would get it filled till the end of the page. Or, for dotted pattern, there are three other parameters — `dotheightfull`, `dotwidthfull`, and `dotwidthtwothirds`. Please, try tweaking those

**Reply by harireghu** [reply]
Thank you! I will take a look and make  the changes.
Will you accept pull requests with the change so other can benefit and I also do not loose it ?

**Reply by kudrykv** [reply]
Yup
